README.md

```diff
@@ -88,4 +88,4 @@ Please keep in mind that this method may lead to a loss in performance, due to t
 * `bundle exec rails about` Show stats on current Rails installation, including version numbers
 
 ### Generating
-* `rails g migration DoSomething` Create migration _db/migrate/*_DoSomething.rb_.
+* `rails g migration DoSomething` Create migration _db/migrate/*_DoSomething.rb_.
```

Comment:
what has been changed here? if there is no change should it really be updated?

Reply:
i think i can remove it